README.md

```diff
@@ -36,6 +36,7 @@ but likely needed for production and test deployments.
 | POSTGRES_ADMIN_PASSWORD   | PosgreSQL database admin password                                                                                   | None                            |
 | REDIS_HOST       | Redis database host user. This system will only function if this variable is defined. (Recommended for production mode)                                                                                       | None                            |
 | REDIS_PASSWORD   | Redis database password                                                                                   | None                            |
+| SUPPORTING_REVOCATION | Enables support for revocation features (true/false)                     | false                           |
```

Comment:
This should not be a global option. Revocation support is a parameter for credential type (add it to the CredentialTypeDto along with name, attributes, etc.)

Reply:
fix: deleted environment variable